public/assets/transition-page/styles/transition-page.css

```diff
@@ -143,3 +145,171 @@ img {
     align-items: center;
   }
 }
+
+/* New transition page stuff */
+
+.masthead-root {
+  padding: 6px 30px;
+}
+
+.top-half {
+  background: #384a51;
+  height: 55vh;
+  min-height: 380px;
+  position: relative;
+  display: flex;
+  justify-content: center;
+}
+
+.top-half-content {
+  text-align: center;
+  color: #ffffff;
+  position: absolute;
+  bottom: 65px;
+}
+
+.top-half-content h3 {
+  font-weight: 600;
+  margin-bottom: 6px;
+}
+
+.top-half-content p {
+  font-size: 16px;
+  margin-bottom: 24px;
+}
+
+.top-half-content img {
+  width: 100%;
+  max-width: 512px;
+}
+
+.browser-image {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  background-image: url(../images/browser.svg);
+  background-repeat: no-repeat;
+  background-position: center;
+  background-size: contain;
+  max-width: 100%;
+  height: 160px;
+  text-align: start;
+  margin-left: auto;
+  margin-right: auto;
+  min-width: 300px;
+}
+
+.links {
+  width: 100%;
+  height: 140px;
+  padding-top: 88.4px;
+  padding-left: 188px;
+  color: #000000;
+}
+
+.phishing {
+  position: relative;
+  font-style: normal;
+  font-weight: normal;
+  font-size: 13px;
+  line-height: 17px;
+  color: #ffffff;
+  text-decoration-line: underline;
+}
+
+.proceed-button {
+  background: #8ca6ad;
+  box-shadow: 0 0 20px rgba(56, 74, 81, 0.5);
+  border-radius: 40px;
+  color: #ffffff;
+  border: none;
+  padding: 12px 25px;
+  font-size: 14px;
+  font-weight: 500;
+  position: absolute;
+  bottom: 0;
+  left: 50%;
+  transform: translate(-50%, 50%);
+  cursor: pointer;
+  display: flex;
+  outline: none;
+}
+
+.proceed-button img {
+  margin-left: 6px;
+}
+
+.bottom-half {
+  text-align: center;
+  position: absolute;
+  bottom: 70px;
+  left: 50%;
+  transform: translateX(-50%);
```

Comment:
Does this work on IE11?

Reply:
pushing to stage to test now, update in 15 min